### DISCUSSION TITLE
capelin should only be modelled for ages 1-4?

### DISCUSSION THREAD

**Discussion by vtrijoulet** [NOAA-EDAB/ms-keyrun#82] [Q&A] [answered]
There are 5 ages for capelin in the surveys but the numbers at age 5 are all 0 for both surveys (simSurveyAgecomp). Not sure why the age 5 is there, above all when there is no weight at age in surveys for age 5 (simSurveyWtatAge). The fishery only catches ages 1-4. Shouldn't the stock be only modelled for ages 1-4 then?

**Comment by vbartolino** [comment, marked as answer]
@vtrijoulet I agree given the data there's no point to have ages >4 for CAP in the models
